package.json

```diff
@@ -44,16 +46,38 @@
   },
   "devDependencies": {
     "async": "^1.5.0",
+    "autoprefixer": "^6.1.1",
     "babel-eslint": "^4.1.5",
+    "babel-loader": "^6.2.0",
     "beautify-lint": "^1.0.4",
     "diff": "^2.2.1",
     "eslint": "^1.9.0",
     "eslint-plugin-babel": "^2.1.1",
     "eslint-plugin-nodeca": "^1.0.3",
     "eslint-plugin-react": "^3.8.0",
+    "express": "^4.13.3",
+    "extract-text-webpack-plugin": "^0.9.1",
     "glob": "^6.0.1",
+    "html-webpack-plugin": "^1.7.0",
     "js-beautify": "^1.5.10",
+    "karma": "^0.13.15",
+    "karma-chrome-launcher": "^0.2.1",
+    "karma-firefox-launcher": "^0.1.7",
+    "karma-phantomjs-launcher": "^0.2.1",
+    "karma-sourcemap-loader": "^0.3.6",
+    "karma-tap": "^1.0.3",
+    "karma-tape-reporter": "^1.0.3",
+    "karma-webpack": "^1.7.0",
     "mocha": "^2.3.3",
-    "webpack-dev-server": "^1.12.1"
+    "node-sass": "^3.4.2",
+    "phantomjs": "^1.9.19",
+    "postcss-loader": "^0.8.0",
+    "react-hot-loader": "^1.3.0",
```

Comment:
[`react-hot-loader`](https://github.com/gaearon/react-hot-loader) has been deprecated in lieu of [babel-plugin-react-transform](https://www.npmjs.com/package/babel-plugin-react-transform) and [react-transform-hmr](https://www.npmjs.com/package/react-transform-hmr), and is the preferred solution for hotloading moving forward.